examples/catalog-module/index.js

```diff
@@ -1,13 +1,20 @@
-import {render} from '../../src/index';
+import React from 'react';
+import ReactDOM from 'react-dom';
+import {Catalog} from '../../src/index';
 
-render({
-  title: 'My Components',
-  pages: [
-    {
-      path: '/',
-      title: 'Foo',
-      imports: {Foo: require('./components/Foo/Foo')},
-      component: require('./components/Foo/Foo.docs.md')
-    }
-  ]
-}, document.getElementById('app'));
+ReactDOM.render(
```

Comment:
I kinda agree; it's definitely an advanced pattern. But removing it would be a breaking change, it's how we embed Catalog in most of our apps. 

But it doesn't expose any mechanism, just gives you some route config objects.

And your example wouldn't work 😬. a) you can't pass a React element to `component` and b) you can't combine routes that way (a component can't add more child routes). What this would do in theory is instantiate another router. It is actually pointless to nest it in an existing route definition. For that to work properly and as expected you need `configureRoutes`.